scss/_buttons.scss

```diff
@@ -131,7 +131,7 @@
   color: #900;
 
   &:hover {
-    color: #fff;
+    color: $text-white;
     background-color: #b33630;
```

Comment:
We have a ton of random colors, I'm not really sure if I should just catalog them all and turn them into variables in this pull.

That may be out of scope for this PR. I was hoping to nail down what we wanted as colors, and then in separate pull requests try and shift some of the colors to be close approximations of what we have. That was just how I was planning on tackling it.